### PR TITLE
Updating ObsData.plot_timeseries() method to use openghg.plotting.plot_timeseries() function

### PR DESCRIPTION
**Pull request by rt17603**
Updating the relevant method on ObsData to use the plot_timeseries() function directly. This is to ensure optimisation and changes don't have to be made twice.

See comments on PR #614

### CODE REVIEW

File: openghg/dataobjects/_obsdata.py [hunk 1]
```diff
@@ -84,46 +84,18 @@ def plot_timeseries(
...
-        font = {"size": 14}
-
-        title_layout = {"text": title, "y": 0.9, "x": 0.5, "xanchor": "center", "yanchor": "top"}
-
-        layout = go.Layout(
-            title=title_layout,
-            xaxis=dict(title=xlabel),
-            yaxis=dict(title=ylabel),
-            font=font,
+        import openghg.plotting
```
Comment (gareth-j) [inline openghg/dataobjects/_obsdata.py:90]: This feels pedantic but we usually do `from openghg.plotting import plot_timeseries`, be good to keep our import style consistent.

**Review by gareth-j — CHANGES_REQUESTED** [review]
Update to keep import style consistent then it'll be ready to merge.

File: openghg/dataobjects/_obsdata.py (hunk 1 above)
Comment (rt17603) [reply openghg/dataobjects/_obsdata.py:90]: Yeah that's fine - my only concern was namespacing as then we'd have two `plot_timeseries` functions and wasn't sure if that would potentially cause a problem? But happy to change it to be consistent if that isn't the case.

Comment (rt17603) [reply openghg/dataobjects/_obsdata.py]: Ok so I've updated it to:
```from openghg.plotting import plot_timeseries as general_plot_timeseries```
to make sure it's clear where this is coming from.

Is that consistent enough with the previous style?

**Review by gareth-j — APPROVED** [review]

File: openghg/dataobjects/_obsdata.py (hunk 1 above)
Comment (gareth-j) [reply openghg/dataobjects/_obsdata.py]: Yeah looks good

**Pull request merged**